pkg/controller/clusterdeployment/clusterdeploymet_controller_test.go

```diff
@@ -70,6 +70,17 @@ func TestReconcile(t *testing.T) {
 			},
 			setupGCPMock: func(r *mockGCP.MockClientMockRecorder) { gomock.Any() },
 		},
+		{
+			name:        "get orgParentFolderID from configmap",
+			expectedErr: nil,
+			localObjects: []runtime.Object{
+				builders.NewTestClusterDeploymentBuilder().GetClusterDeployment(),
+				builders.NewTestConfigMapBuilder(orgGcpSecretName, operatorNamespace, "111111").GetConfigMap(),
+				builders.NewTestSecretBuilder(orgGcpSecretName, operatorNamespace, "testCreds").GetTestSecret(),
+				builders.NewTestSecretBuilder(gcpSecretName, testNamespace, "testCreds").GetTestSecret(),
+			},
+			setupGCPMock: func(r *mockGCP.MockClientMockRecorder) { gomock.Any() },
```

Comment:
I have no idea what this does, and how this tests your code. I.e. where does it test what you put into the `ConfigMap` is actually digged out by `GetGCPParentFolderFromConfigMap`?

Reply:
this test is fixed for testing negative scenario: if the configmap does not contain orgParentFolderID